### PR TITLE
Hubert / Update index scss

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: src/components/homepage/WaysToEarn/WaysToEarn.module.scss [hunk 1]
```diff
@@ -57,12 +48,3 @@
         }
     }
 }
```
Comment (maryia-binary) [inline src/components/homepage/WaysToEarn/WaysToEarn.module.scss:50]: @hubert-deriv nit: please add a line break at the last line😊

File: src/components/homepage/HomepageSlider/HomepageSlider.module.scss [hunk 2]
```diff
@@ -3,16 +3,54 @@
...
+  }
+
+  .prev:active,
+  .next:active {
+    transform: scale(0.8);
+  }
+
   .sliderWrapper {
     width: 33vw;
+    height: 248px;
```
Comment (maryia-binary) [inline src/components/homepage/HomepageSlider/HomepageSlider.module.scss:41]: @hubert-deriv I would suggest replacing `px` with `rem` for the newly added styles where applicable and depending on how much pixels there is in 1 rem in this project 🙂

File: src/components/homepage/HomepageFooter/HomepageFooter.module.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.communityBtn {
+    padding: 6px 16px;
+    border-radius: 4px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 20px;
+    border: solid 1px var(--gray-04);
+}
```
Comment (maryia-binary) [inline src/components/homepage/HomepageFooter/HomepageFooter.module.scss:8]: nit: just a missing line break )

File: src/components/global/InlineCode/InlineCode.module.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.inlineCode {
+    font-weight: 400;
+    color: var(--red-01);
+    border-radius: 4px;
+    padding: 2px 4px;
+    background-color: var(--gray-01);
+    border: 1px solid var(--gray-02);
+  }
```
Comment (maryia-binary) [inline src/components/global/InlineCode/InlineCode.module.scss:8]:
```suggestion
  }
  
```

File: src/components/api-guide/ApiGuide/DerivGuideContent/DerivGuideContent.tsx [hunk 5]
```diff
@@ -2,6 +2,7 @@ import React from "react"
 import { Link } from "react-router-dom";
 import ApiGuideTable from "../ApiGuideTable/ApiGuideTable";
 import CodeBlockSingleLanguage from "../../../global/CodeBlock/CodeBlockSingleLanguage";
+import { InlineCode } from "../../../global/InlineCode/InlineCode";
```
Comment (maryia-binary) [inline src/components/api-guide/ApiGuide/DerivGuideContent/DerivGuideContent.tsx:5]:
```suggestion
import { InlineCode } from "global/InlineCode/InlineCode";
```
could you please check if you have config for the project with src directory set directly in order to put the file path starting from an src subfolder? or could you maybe check if this works?🙂

File: src/components/homepage/Benefits/Benefits.module.scss [hunk 6]
```diff
@@ -44,6 +36,10 @@
         justify-content: space-evenly;
         width: 100%;
 
+        h3 {
+            margin: 16px 0;
+        }
```
Comment (yauheni-kryzhyk-deriv) [inline src/components/homepage/Benefits/Benefits.module.scss:41]: so we start using rem? if so, should we change px to rem here?

File: src/components/homepage/ClientLibraries/ClientLibraries.module.scss [hunk 7]
```diff
@@ -1,72 +1,71 @@
-.clientLibrary {
-    .iconJsLibrary {
-        background-image: url(/img/js-library.svg);
-        background-repeat: no-repeat;
-        background-position: var(--text-align-center);
-        width: 6.4rem;
-        height: 6.4rem;
-    }
+.takeToLib {
+    padding: 80px 0 0;
```
Comment (yauheni-kryzhyk-deriv) [inline src/components/homepage/ClientLibraries/ClientLibraries.module.scss:2]: here also the same question about rem

File: src/components/homepage/GetStarted/GetStarted.module.scss [hunk 8]
```diff
@@ -17,4 +17,20 @@
...
+}
+
+[data-state*="responsive.mobile"] {
+
+    .termConditions,
+    .termConditions a {
+        text-align: center;
+        max-width: 350px;
+        font-size: 13px;
+        margin-top: 0;
```
Comment (yauheni-kryzhyk-deriv) [inline src/components/homepage/GetStarted/GetStarted.module.scss:34]: and here also

File: src/components/homepage/HomepageFooter/HomepageFooter.module.scss [hunk 9]
```diff
@@ -0,0 +1,8 @@
+.communityBtn {
+    padding: 6px 16px;
```
Comment (matin-binary) [inline src/components/homepage/HomepageFooter/HomepageFooter.module.scss:2]: Need to update most of the values in this file.
- px to rem
- use predefined font-sizes

File: src/components/homepage/HomepageSlider/HomepageSlider.module.scss [hunk 10]
```diff
@@ -3,16 +3,54 @@
   left: 50%;
   transform: translate(0);
   width: 33vw;
-  height: 24.8rem;
+  height: 248px;
```
Comment (matin-binary) [inline src/components/homepage/HomepageSlider/HomepageSlider.module.scss:6]: ditto

File: src/components/homepage/HomepageSlider/HomepageSlider.module.scss [hunk 11]
```diff
@@ -3,16 +3,54 @@
...
+
+  .control {
+    position: absolute;
+    top: 35%;
+    width: 50px;
```
Comment (matin-binary) [inline src/components/homepage/HomepageSlider/HomepageSlider.module.scss:11]: ditto

File: src/index.scss [hunk 12]
```diff
@@ -593,245 +138,249 @@ textarea {
...
+    line-height: 1.25;
+  }
 
-.main-page-card .header {
-  display: flex;
-  width: 100%;
-}
+  >h2 {
+    font-size: var(--text-size-l);
+    margin-bottom: 8px;
```
Comment (matin-binary) [inline src/index.scss:235]: ditto

File: src/index.scss [hunk 13]
```diff
@@ -593,245 +138,249 @@ textarea {
...
-  display: flex;
-  flex-direction: column;
-  height: fit-content;
-  flex-wrap: wrap;
-  align-items: center;
-  justify-content: center;
-  width: 80%;
+  max-width: 316px;
+  padding: 3.2rem 2.4rem;
+  border-radius: 4px;
```
Comment (matin-binary) [inline src/index.scss:152]: ditto

File: src/index.scss [hunk 14]
```diff
@@ -110,263 +6,61 @@ body {
...
   }
 }
 
-li {
+textarea {
   font-size: var(--text-size-s);
-  margin-top: 16px;
+  resize: none;
+  width: calc(100% - 1.6rem);
+  min-height: 119px;
```
Comment (matin-binary) [inline src/index.scss:59]: ditto

File: src/index.scss [hunk 15]
```diff
@@ -110,263 +6,61 @@ body {
...
-    left: -50px;
-  }
-
-  .main-content .main-page-row .column-container .next {
-    right: -50px;
+    &::before {
+      content: "";
+      display: inline-block;
+      position: absolute;
+      height: 40px;
```
Comment (matin-binary) [inline src/index.scss:18]: ditto

File: src/components/api-guide/ApiGuide/DerivGuideContent/DerivGuideContent.tsx (hunk 5 above)
Comment (hubert-deriv) [reply src/components/api-guide/ApiGuide/DerivGuideContent/DerivGuideContent.tsx:5]: Ah. this folder is actually not set in the root of the project. It is inside src > components > global. We have not a config set for this project to direct to a certain folder.

File: src/components/app-registration/AppRegistration/AppRegistration.scss [hunk 16]
```diff
@@ -166,9 +166,9 @@
   background-color: var(--white);
   color: black;
   font-weight: var(--text-weight-bold);
-  border: 1px solid #999999;
+  border: 0.1rem solid #999999;
```
Comment (matin-binary) [inline src/components/app-registration/AppRegistration/AppRegistration.scss:169]: - Borders don't need to scale.
- Can replace color with a var
```suggestion
  border: 1px solid var(--gray-05);
```

File: src/components/app-registration/AppRegistration/AppRegistrationForm/AppRegistrationForm.module.scss [hunk 17]
```diff
@@ -4,8 +4,8 @@ fieldset .customTextInput:last-child {
 
 .customTextInput {
     align-items: center;
-    border: 1px solid var(--border-normal);
-    border-radius: 4px;
+    border: 0.1rem solid var(--border-normal);
+    border-radius: 0.4rem;
```
Comment (matin-binary) [inline src/components/app-registration/AppRegistration/AppRegistrationForm/AppRegistrationForm.module.scss:8]: Help to revert all the border changes, logically, font-size changes should not scale border/border-radius sizes.

File: src/components/global/InlineCode/InlineCode.module.scss [hunk 18]
```diff
@@ -0,0 +1,9 @@
+.inlineCode {
+    font-weight: 400;
+    color: var(--red-01);
+    border-radius: 4px;
+    padding: 0.2rem 0.4rem;
+    background-color: var(--gray-01);
+    border: 1px solid var(--gray-02);
+  }
+
```
Comment (maryia-binary) [inline src/components/global/InlineCode/InlineCode.module.scss:9]: @hubert-deriv nit: unnecessary extra spaces 🙂
```suggestion

```

File: src/components/global/Accordion/Accordion.module.scss [hunk 19]
```diff
@@ -1,79 +1,85 @@
...
+}
+
+[data-state*="responsive.desktop"] {
+  .accordion {
+    padding-left: 7.5rem;
+    .accordionHeader {
+      padding: 1.6rem 2.8rem;
+    }
+  }
+}
```
Comment (maryia-binary) [inline src/components/global/Accordion/Accordion.module.scss:85]:
```suggestion
}

```
nit

**Review by maryia-binary — APPROVED** [review]

**Review by yauheni-kryzhyk-deriv — APPROVED** [review]

**Review by matin-binary — APPROVED** [review]

**Pull request merged**